apim/templates/api/api-configmap.yaml

```diff
@@ -114,11 +114,13 @@ data:
           # - none : passwords are not hashed/encrypted
           # default value is bcrypt
           password-encoding-algo: bcrypt
+          allow-email-in-search-results: {{ .Values.inMemoryAuth.allowEmailInSearchResults }}
           users:
             - user:
               username: admin
               password: {{ .Values.adminPasswordBcrypt }}
               roles: MANAGEMENT:ADMIN, PORTAL:ADMIN
+              email: {{ .Values.adminEmail }}
```

Comment:
Do you think that it makes sense here to set the email if `adminEmail` is empty?

Reply:
No, but this is the default value and adding a condition to just ignore it will reduce readability and have the same effect, so I preferred adding it whatever the value. Would you prefer me to add the condition?

Reply:
Nope, it was just to be sure

Reply:
So do you need further changes to approve this PR?